pages/common/git-missing.md

```diff
@@ -0,0 +1,13 @@
+# git missing
+
+> Show commits which aren't shared between two branches.
```

Comment:
```suggestion
> Show commits which are not shared between two branches.
```

`aren't` does not feel formal

Reply:
aren't is more than acceptable, in fact I would argue `are not` is incorrect.
Also although its only a few, `are not` is used 8 files while `aren't` has 11 files

Reply:
Should we be consistent and change are not to aren't in those pages as well ?

Reply:
Whats wrong with aren't, its better, reads nicer

Reply:
It's merely a matter of preference.  I personally prefer `are not` here because of the formality (`aren't` sounds just a tad too long and informal to me).

>in fact I would argue `are not` is incorrect.

_That_ is incorrect — `aren't` is the contraction of `are` and `not`; thus, `are not` and `aren't` are identical.

Reply:
>Should we be consistent and change are not to aren't in those pages as well ?

It's not really something that ought to be standardized.  I don't think we should do that.

>Also although its only a few, are not is used 8 files while aren't has 11 files

Those numbers are too small to be meaningful :)

Reply:
aren't definitely reads shorter than are not

Reply:
Yes I agree, it's merely preference. I think this is a nitpick.
Personally this felt informal, but looks like more files have 'aren't'.

Reply:
>aren't definitely reads shorter than are not

You're right — the was I was pronouncing it in my head sound longer (`arnt` vs `arent`) 🙂

Reply:
>Personally this felt informal, but looks like more files have 'aren't'.

As I said before, those numbers are _so_ small (and similar) that it's not a good idea to base decisions of them.

Reply:
so what are we doing here

Reply:
Let's keep it as `aren't`, because I don't really see why we should change it.